index.js

```diff
@@ -35,6 +36,7 @@ module.exports.filter = shouldCompress
  */
 
 var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
+var nodeHasCallbacks = (http.OutgoingMessage.prototype.write.length === 3 && http.OutgoingMessage.prototype.end.length === 3)
```

Comment:
And that comment was in response to the following change: https://github.com/expressjs/compression/pull/80/commits/0f617c7f15a1826adf3d8782b282facd6cac4642 which is basically identical to your change here.